### ISSUE TITLE
How to send text input from user in realtime api? Docs aren't clear

### ISSUE THREAD

**Issue by bags307** [livekit/agents-js#193]
I am completely confused.  How does one send text to the realtime API.  I understand how to using the raw Openai Realtime api, but can't get it working via livekit.

**Comment by skye0402** [comment]
Good point - looking at the code I think text-only is not supported? But this would be perfect for testing as the audio is very costly! Any input from the maintainers?

**Comment by nbsp** [comment]
we don't currently support text-only mode. however, it isn't necessary to use our OpenAI plugin in order to use the Realtime API within Agents — you can simply use Agents to orchestrate the jobs and then call OpenAI's API directly via the Realtime SDK.

**Comment by bags307** [comment]
@nbsp can you expand on what you mean?

**Comment by smrpl6** [comment]
@bags307 did you manage to make it work ? I am facing the same issue with unclear documentation on how to send text message from client and handle it with multi modal (text & audio) openai realtime api
